leveller.js

```diff
@@ -61,16 +61,24 @@
     var diff = targetHeight - currentHeight;
     if (diff === 0) return;
 
+    var isHeightProperty = (this.options.cssProperty.indexOf('eight') > 0);
```

Comment:
Possible values I need to check for are:

- `height`
- `min-height`
- `max-height` (weird, but possible)
- `minHeight`
- `maxHeight`

And any other lowercase, uppercase or untrimmed version of those options.

After doing some digging, I found that doing an `indexOf` check was faster than using regex or normalizing the string and looping through an array of acceptable values.